imqsauth/perms.go

```diff
@@ -30,6 +30,7 @@ const (
 	PermPCSBudgetItemsAvailable   authaus.PermissionU16 = 116 // User is allowed to view the allocatable budget items
 	PermReportCreator             authaus.PermissionU16 = 200 // Can create reports
 	PermReportViewer              authaus.PermissionU16 = 201 // Can view reports
+	PermImporter                  authaus.PermissionU16 = 500 // User is allowed to handle data imports
```

Comment:
Unless you have a good reason, rather start this new group at 300, instead of 500.